concourse/templates/sap-test.libsonnet

```diff
@@ -0,0 +1,150 @@
+{
+  saptestjob:: {
+    local tl = self,
+
+    image:: error 'must be set',
+
+    name: 'sap-workload-test-' + self.image,
+
+    plan: [
+    {
+      get: tl.image + '-gcs',
+      passed: [
+        'publish-to-testing-' + tl.image,
+      ],
+      params: {
+        skip_download: 'true',
+      }
+    },
+    { get: 'guest-test-infra' },
+    {
+      task: 'generate-timestamp',
+      file: 'guest-test-infra/concourse/tasks/generate-timestamp.yaml',
+    },
+    {
+      load_var: 'id',
+      file: 'timestamp/timestamp-ms',
+    },
+    {
+      task: 'generate-post-script',
+      config: {
+        platform: 'linux',
+        image_resource: {
+          type: 'registry-image',
+          source: {
+            repository: 'google/cloud-sdk',
+            tag: 'latest',
+          },
+        },
+        inputs: [
+          { name: "guest-test-infra" },
+        ],
+        run: {
+          path: 'sh',
+          args: [
+            '-exc',
+            |||
+            cd guest-test-infra/concourse/workload-tests/SAP/
```

Comment:
1. this could also be replaced by setting `path:` in the input
2. let's put the scripts in `concourse/scripts` not `concourse/workload-testing/SAP`